packages/wonder-blocks-popover/src/components/popover.tsx

```diff
@@ -255,32 +258,44 @@ export default class Popover extends React.Component<Props, State> {
     }
 
     renderPopper(uniqueId: string): React.ReactNode {
-        const {initialFocusId, placement, showTail} = this.props;
+        const {initialFocusId, placement, showTail, disablePortal} = this.props;
         const {anchorElement} = this.state;
 
-        return (
-            // Ensures the user is focused on the first available element
-            // when popover is rendered
-            <InitialFocus initialFocusId={initialFocusId}>
-                <TooltipPopper
+        const popperContent = (
+            <TooltipPopper anchorElement={anchorElement} placement={placement}>
+                {(props: PopperElementProps) => (
+                    <PopoverDialog
+                        {...props}
+                        aria-describedby={`${uniqueId}-content`}
+                        aria-labelledby={`${uniqueId}-title`}
+                        id={uniqueId}
+                        onUpdate={(placement) => this.setState({placement})}
+                        showTail={showTail}
+                    >
+                        {this.renderContent(uniqueId)}
+                    </PopoverDialog>
+                )}
+            </TooltipPopper>
+        );
+
+        if (disablePortal) {
+            return (
+                // Ensures the user is focused on the first available element
+                // when popover is rendered without the focus manager.
+                <InitialFocus initialFocusId={initialFocusId}>
+                    {popperContent}
+                </InitialFocus>
+            );
+        } else {
+            return (
+                <FocusManager
                     anchorElement={anchorElement}
-                    placement={placement}
+                    initialFocusId={initialFocusId}
                 >
-                    {(props: PopperElementProps) => (
-                        <PopoverDialog
-                            {...props}
-                            aria-describedby={`${uniqueId}-content`}
-                            aria-labelledby={`${uniqueId}-title`}
-                            id={uniqueId}
-                            onUpdate={(placement) => this.setState({placement})}
-                            showTail={showTail}
-                        >
-                            {this.renderContent(uniqueId)}
-                        </PopoverDialog>
-                    )}
-                </TooltipPopper>
-            </InitialFocus>
-        );
+                    {popperContent}
+                </FocusManager>
+            );
+        }
```

Comment:
**suggestion:** The `else` here could be removed here as the if stops at the return in line 290.

e.g.
```
if () {

}

return (<FocusManager....
```

Reply:
Very true, I just enjoy being explicit with my return statements in an if/else, though you are correct it's not strictly neccesary. Let me know if you would prefer I remove the else.